app/hotels/src/map/singleHotel/SingleHotelMapScreen.js

```diff
@@ -61,19 +63,18 @@ class SingleHotelMapScreen extends React.Component<Props> {
     );
   };
 
-  renderHotelsContext = ({ currency, roomsConfiguration }) => (
-    <SingleHotelContext.Consumer>
-      {this.renderQuery(currency, roomsConfiguration)}
-    </SingleHotelContext.Consumer>
-  );
-
   render() {
     return (
-      <HotelsContext.Consumer>
-        {this.renderHotelsContext}
-      </HotelsContext.Consumer>
+      <SingleHotelContext.Consumer>
```

Comment:
Could this context not be extracted too?

Reply:
Step by step, I am also considering merging those two contexts into onw